resources/migrations/051_add_new_query_execution_table.yaml

```diff
@@ -0,0 +1,82 @@
+databaseChangeLog:
+  - changeSet:
+      id: 51
+      author: camsaul
+      changes:
+        - createTable:
+            tableName: query_execution
+            remarks: 'A log of executed queries, used for calculating historic execution times, auditing, and other purposes.'
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: hash
```

Comment:
👍 
There's a bit of weirdness in that we might want to examine the query as it was when the user ran it (ie. use the version history of a question), but let's burn that bridge when we come to it.